### DISCUSSION TITLE
Apktool not able to build the payload binded application

### DISCUSSION THREAD

**Discussion by Rllymn7** [termux/termux-packages#9033] [Q&A]
I: Using Apktool 2.3.1
I: Checking whether sources has changed...
I: Smaling smali folder into classes.dex...
I: Checking whether resources has changed...
I: Building resources...
Exception in thread "main" W: /data/data/com.termux/files/usr/tmp/brut_util_Jar_14794345089913304098.tmp[57]: @8: inaccessible or not 

![Screenshot_20220214-232619_Termux.jpg](https://user-images.githubusercontent.com/99534061/153919866-00a4dd65-8935-43c0-b1b7-bafc70628fc0.jpg)

**Comment by xtkoba** [comment]
Upstream: https://github.com/iBotPeaches/Apktool/issues/2092. Not going to be fixed unfortunately.
